test/scrabble_test.rb

```diff
@@ -6,8 +6,30 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
+    #skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_it_can_score_a_word
+    assert_equal 2, Scrabble.new.score("an")
+    assert_equal 6, Scrabble.new.score('dim')
+  end
+
+  def test_it_scores_empty_string_as_zero_points
+    assert_equal 0, Scrabble.new.score('')
+  end
+
+  def test_it_scores_nil_input_as_zero_points
+    assert_equal 0, Scrabble.new.score('')
+  end
```

Comment:
This one doesn't quite test nil input. You might pass nil as the argument or pass no argument at all with the variable being defaulted to nil.